webserver/__init__.py

```diff
@@ -31,6 +33,9 @@ def create_app():
     from kafka_connection import init_kafka_connection
     init_kafka_connection(app.config['KAFKA_CONNECT'])
 
+    # Redis connection
+    app.redis = Redis()
```

Comment:
Wasn't there a problem in london related to this? I forget. 

However, as far as I understand you can assign things to the app until you handle the first request -- then you're not allowed anymore.